CHANGES

```diff
@@ -4,6 +4,13 @@
 
  Changes between 1.0.2g and 1.1.0  [xx XXX xxxx]
 
+  *) Unify TYPE_up_ref(obj) methods signature.
+     SSL_CTX_up_ref(), SSL_up_ref(), X509_up_ref(), EVP_PKEY_up_ref(), 
+     X509_CRL_up_ref(), X509_OBJECT_up_ref_count() methods are now returning an
+     int (instead of void) like all others TYPE_up_ref() methods.
+     So now these methods also check the return value of CRYPTO_atomic_add(), 
+     and the validity of object reference counter.
```

Comment:
Nit: Normally you would add your name in square brackets underneath the CHANGES entry (although I note the one below this doesn't have that either....but all the others do!).